blocks/centered-content-switcher/centered-content-switcher.js

```diff
@@ -60,7 +60,9 @@ export default function decorate(block) {
     ul(
       { class: 'desktop-nav-list' },
       ...[...block.children].map((child, idx) => {
-        const thumbnailUrl = child.children[0].children[2].querySelector('img').src;
+        const thumbnailUrl = child.children[0].children[3].querySelector('img').src;
```

Comment:
As the picture/img is always the first one within the div, why not just select child.children[0].querySelector('picture')? Then if you place it elsewhere you don't need to worry about removing it (or the parent picture).